db_medical_store.sql

```diff
@@ -197,8 +197,8 @@ create table t_approvisionnement
 	code_produit nvarchar(50),
 	code_fournisseur nvarchar(50),
 	code_depot nvarchar(50),
-    ugs nvarchar ----unite de gestion de stock, milligrammes
-	quantite int,
+    ugs nvarchar, ----unite de gestion de stock, milligrammes
+	quantite float,
```

Comment:
Ici je ne vois pas pourquoi tu as mis la quantité en float, déjà que les valeurs de boîtes sont souvent exactes, on aura pas de distribution de trucs pas entiers. Tout est integer ici.

Reply:
I'll rechange it into int